docs/CONFIG-VARS.md

```diff
@@ -197,7 +197,7 @@ Custom policy:
 | <div style="width:50px">Name</div> | <div style="width:150px">Description</div> | <div style="width:50px">Type</div> | <div style="width:75px">Default</div> | <div style="width:150px">Notes</div> |
 | :--- | :--- | :--- | :--- | :--- |
 | create_static_kubeconfig | Allows the user to create a provider- or service account-based kubeconfig file | bool | false | A value of `false` defaults to using the cloud provider's mechanism for generating the kubeconfig file. A value of `true` creates a static kubeconfig that uses a service account and cluster role binding to provide credentials. |
```

Comment:
There is a mismatch for `create_static_kubeconfig` default value. The variables.tf file has default set to `true` whereas the CONFIG-VARS states the default to be false.
Could you update CONFIG-VARS to point correct default value?

Reply:
Good catch, fixed the doc.